[storage] Rename NVMe -> Nvme

glimchb opened the pull request:
- [storage] Rename NVMe -> Nvme
- [storage] auto-generate protos

seroyer reviewed (approved):
It would be nice to add something into the commit message for why it is being renamed, so that is recorded in the repo.

artek-koltun reviewed (commented):
We've already renamed NVMe to Nvme some time ago. Here we renamed NVMf to Nvmf. Could we adjust then commit message and PR title/description accordingly?

glimchb commented:
> We've already renamed NVMe to Nvme some time ago. Here we renamed NVMf to Nvmf. Could we adjust then commit message and PR title/description accordingly?

@artek-koltun you are right. nvmf is stupid. let's call it nvme instead. pushed an update. this will also help us consolidate nvme on the FE and BE going forward I think, because they really start to look similar

glimchb commented on storage/v1alpha1/backend_nvme_tcp.proto:
```diff
@@ -326,11 +326,11 @@ enum NvmeTransportType {
 
 enum NvmeAddressFamily {
     NVME_ADDRESS_FAMILY_UNSPECIFIED = 0;
-    NVMF_ADRFAM_IPV4                = 1;
```
see ne pushed commit

artek-koltun commented:
> @artek-koltun you are right. nvmf is stupid. let's call it nvme instead. pushed an update. this will also help us consolidate nvme on the FE and BE going forward I think, because they really start to look similar

Ok, let's unify naming to Nvme form only. I don't mind.
However, I still see commit message for 8d0ca21 as
```
[storage] Rename NVMe -> Nvme

Naming NVMe caused proto to force "nv_me" syntax which is weird
```

Let's update it to better represent the point of the change, maybe smth like
```
[storage] Rename NVMf -> Nvme

Naming NVMf caused proto to force "nv_mf" syntax which is weird.
Unify the naming to Nvme format utilized throughout all API components.
```

Wdyt?

glimchb commented:
> Wdyt?

done
